lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
Enabling this disables site generation, but this probably shouldn't be the case.
Perhaps it would be sensible to add a --dry-run flag as well that disables site generation? Then the invocation would be:

jazzy --lint --dry-run